crates/ty/src/args.rs

```diff
@@ -82,10 +82,11 @@ pub(crate) struct CheckCommand {
     /// type definitions of first- and third-party modules that are conditional on the Python version.
     ///
     /// By default, the Python version is inferred as the lower bound of the project's
-    /// `requires-python` field from the `pyproject.toml`, if available. Otherwise, if a virtual
-    /// environment has been configured or detected and a Python version can be inferred from the
-    /// virtual environment's metadata, that version will be used. If neither of these applies, ty
-    /// will fall back to the latest stable Python version supported by ty (currently 3.13).
+    /// `requires-python` field from the `pyproject.toml`, if available. Otherwise, if a Python
+    /// installation has been configured or detected and a Python version can be inferred from the
+    /// metadata or layout of that Python installation, that version will be used. If neither of
+    /// these applies, ty will fall back to the latest stable Python version supported by ty
+    /// (currently 3.13).
     #[arg(long, value_name = "VERSION", alias = "target-version")]
```

Comment:
We should also update the `EnvironmentOptions::python_version` documentation. 

The sentence here is somewhat complicated and I don't really feel wiser after reading other than: Okay, ty does some magic but I don't have good suggestions on how to make it clearer (other than trying to split the sentence into multiple sentences to avoid the many and/or)